web/src/components/onboarding/OnboardingEssentialQuestion.tsx

```diff
@@ -48,7 +48,7 @@ export const OnboardingEssentialQuestion = <T,>(props: Props<T>): ReactElement =
         />
         {isFormFieldInValid && (
           <div className="text-error-dark text-bold margin-top-05">
-            {Config.profileDefaults.essentialQuestionInlineText}
+            {Config.profileDefaults.default.essentialQuestionInlineText}
```

Comment:
Did you consider putting the variables top-level instead of nested inside the `default` object? Then, all these changes would not be necessary because the path would remain the same.

Reply:
Good call Anne!

Reply:
Yes, the thought did come up. I decided to go this route because I wanted to store the data in a consistent manner. The business formation JSON file was one file that I look that closely, it has nested objects and does not store variables are the top level.